4_Bayes_Rule/Bayes_Rule_All.ipynb

```diff
@@ -0,0 +1,4120 @@
+{
```

Comment:
<ul><li>Maybe add that there is no "one best prior" -- it's not Matrix: the One doesn't exist</li><li>Potential for a joke here. I'd try something like: "I don't remember if it's George Box or one of the Kardashians who said: "All models are wrong, but some are useful"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>